src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -84,8 +87,10 @@ def is_installed(cls):
             HAVE_MS5 = True
         except ImportError:
             HAVE_MS5 = False
+            mountainsort5 = None
```

Comment:
```suggestion
```
@magland any reason to add this and line 93?? 
If `mountainsort5` fails to import, it will never get to line 93

Reply:
it's to keep pyright linter happy.